addons/actions/src/manager.tsx

```diff
@@ -5,13 +5,9 @@ import { ADDON_ID, PANEL_ID } from '.';
 
 export function register() {
   addons.register(ADDON_ID, api => {
-    const channel = addons.getChannel();
     addons.addPanel(PANEL_ID, {
       title: 'Actions',
-      // eslint-disable-next-line react/prop-types
-      render: ({ active, key }) => (
-        <ActionLogger key={key} channel={channel} api={api} active={active} />
-      ),
+      render: ({ active, key }: any) => <ActionLogger key={key} api={api} active={active} />,
```

Comment:
Do we not know the types of `active` and `key` here?

Reply:
`active` must be a `boolean` and `key` looks not to be used in `ActionLogger` component so I will remove it.

Reply:
Sounds good 😄